house-snark/src/relations/blender/withdraw.rs

```diff
@@ -355,17 +358,9 @@ mod tests {
             recipient,
         );
 
-        let input = [
-            circuit.fee.clone(),
-            circuit.recipient.clone(),
-            circuit.token_id.clone(),
-            circuit.old_nullifier.clone(),
-            circuit.new_note.clone(),
-            circuit.token_amount_out.clone(),
-            circuit.merkle_root.clone(),
-        ];
+        let input = circuit.public_input();
```

Comment:
So there's a trait that is implemented for an array of elements?

Also, why not 
```rust
        let input = circuit.public_input().try_into().unwrap())
```
right away?

Reply:
what would be the advantage, because AFAIK the two are equivalent?